tests/testrenderclass.py

```diff
@@ -205,7 +205,7 @@ def test_get_size_grid_plot(self):
         grid = GridSpace({(i, j): self.image1 for i in range(3) for j in range(3)})
         plot = self.renderer.get_plot(grid)
         w, h = self.renderer.get_size(plot)
-        self.assertEqual((w, h), (360, 360))
+        self.assertEqual((w, h), (418, 410))
```

Comment:
Interesting that it used to be perfectly square, but isn't anymore. Generally, square plots seems easier when considering layouts etc...